### PR TITLE
bump morphology-repair-workflow to 2.0.2

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class PyMorphologyRepairWorkflow(PythonPackage):
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/nse/morphology-repair-workflow"
     git      = "ssh://bbpcode.epfl.ch/nse/morphology-repair-workflow"
     version('develop', branch='master')
+    version('2.0.2', tag='morphology-repair-workflow-v2.0.1')
```
Comment (genric) [inline var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py]: Tag label need the last character to be 2.

**Review by genric — CHANGES_REQUESTED** [review]
IGNORE THIS > Please also update `deploy/packages/bbp-packages.yaml` with new version.

File: var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py]: Good catch! The `packages/*.yaml` are superseded by `environments/*.yaml`, and should normally not have any version pins any longer

Comment (genric) [reply var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py]: So we can skip `deploy/packages/bbp-packages.yaml` update even before you'll merge https://github.com/BlueBrain/spack/pull/1013 ?

Comment (genric) [reply var/spack/repos/builtin/packages/py-morphology-repair-workflow/package.py]: Ah, nice :) now I see where the old jupyter stuff is coming from :) `py-jupyter-notebook@4.2.3` I'll rename it with next commit.

**Comment by genric** [comment]
Retest this please

**Review by genric — APPROVED** [review]

**Pull request merged**